### PR TITLE
Implemented Metrics-Plugin

### PR DESCRIPTION
**Pull request by qacwnfq**
I refactored the reporters again to better include the metrics-reporter. All reporters now refer to a common report-paths.js-file for finding the paths. This is important, because if we want html-reports, we have to give the metrics-reporter access to the html-report-path, so the metrics are included in the
interactive html-report.
The metrics-reporter can't have the same api as the other reporters because the metric-items
are not included in the results. This explains metrics behind handed differently from results in src/services/cli.js

### CODE REVIEW

File: src/services/cli.js [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
  * @module cli
  */
-
 /* global fetch */
 require( 'isomorphic-fetch' );
 
 const createLoader = require( './loader' );
 const createRunner = require( './runner' );
 
+const { generateOutputForJenkins } = require( '../reporters/metric.js' );
```
Comment (x1B) [inline src/services/cli.js]: Should not mention "Jenkins" directly, instead we can name the output format (JSON).

**Comment by qacwnfq** [comment]
I've split up the fixes into small commits since the github-interfice allows to squash everything into one commit before mergin.

**Comment by qacwnfq** [comment]
Preview: 
screenshot from 2017-12-21 16-44-26

**Comment by qacwnfq** [comment]
Moved to #8

**Pull request closed without merging**